src/nvim/screen.c

```diff
@@ -6079,8 +6079,10 @@ void grid_alloc(ScreenGrid *grid, int rows, int columns, bool copy, bool valid)
 {
   int new_row;
   ScreenGrid new = *grid;
-
-  size_t ncells = (size_t)((rows+1) * columns);
+  // Since amount of cells could not be negative we expect both
+  // rows and columns to be positive
```

Comment:
the comment is redundant with the  `assert()`

Reply:
removed